catalog/app/components/JsonDisplay/JsonDisplay.js

```diff
@@ -75,18 +78,38 @@ function NonStringValue({ value }) {
   return <div>{`${value}`}</div>
 }
 
+function S3UrlValue({ href, children }) {
+  const { urls } = NamedRoutes.use()
+  const isInStack = BucketConfig.useIsInStack()
+  const props = React.useMemo(() => {
+    const s3Handle = s3paths.parseS3Url(href)
+    const inStack = isInStack(s3Handle.bucket)
+    return {
+      href: inStack
+        ? urls.bucketFile(s3Handle.bucket, s3Handle.key, s3Handle.version)
```

Comment:
internal catalog url should be passed as `to` prop to be handled by react-router

Reply:
Thanks, I didn't notice reload